robyn/router.py

```diff
@@ -124,11 +116,74 @@ def add_route(
         exception_handler: Optional[Callable],
         injected_dependencies: dict,
     ) -> Union[Callable, CoroutineType]:
+        def wrapped_handler(*args, **kwargs):
+            # In the execute functions the request is passed into *args
+            request = next(filter(lambda it: isinstance(it, Request), args), None)
+
+            handler_params = signature(handler).parameters
+
+            if not request or len(handler_params) == 1 and next(iter(handler_params)) is Request:
```

Comment:
@VishnuSanal could you please add braces here?
```suggestion
            if not request or (len(handler_params) == 1 and next(iter(handler_params)) is Request):
```

Does this mean this?